public/amavis-recent-mail.php

```diff
@@ -158,8 +158,16 @@
     $r['mail_id'] = $mail_id;
     $r['base64_message_id'] = $base64_message_id;
 
```

Comment:
This should be an added compatibility, instead of an introduced incompatibility

Reply:
I don't understand the context of your comment. 

(if it's about bas64 encoding of the message_id - It's necessary as I use it in URLs and it was easier to just base64 encode than figure out what ones need escaping/handling (e.g. a message id can contain: < > @ + ) )

Reply:
Sorry maybe misplaced my comment, i also added some stream_get_contents vs simple parameter conditions, that allows the retrieval of sender and recipient.